ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -461,6 +463,28 @@ export default function ViewQuote() {
       : 'ETH',
   ])
 
+  const shouldShowPriceDifferenceWarning =
+    !showInsufficientWarning &&
+    ['high', 'medium'].includes(usedQuote?.priceSlippage?.bucket)
+
+  const priceSlippageFromSource = useEthFiatAmount(
+    usedQuote?.priceSlippage?.sourceAmountInETH || 0,
```

Comment:
I think if either `sourceAmountInETH` or `destinationAmountInEth` is falsy, we should no show the price difference warning. (If the api made a mistake and we defaulted one of these to 0, the user would see a confusing message where it would say that they are send or receiving 0 USD worth of something)

Reply:
Per Sam:

```
No there isn’t a case where that would happen
Because you couldn’t get a non-zero ratio from dividing two numbers where one was falsely
```

Reply:
Or I suppose I should generalize might point: if either are undefined/null, we shouldn't assume that they have 0 fiat value, instead we should just treat that as if we don't know the conversion rate at all. I think that Gal and Jake are developing some copy for that scenario, and we should use that copy in any case where the `sourceAmountInETH` or `destinationAmountInEth` values are null/undefined